ocs_ci/ocs/utils.py

```diff
@@ -1003,6 +1048,41 @@ def collect_prometheus_metrics(
             json.dump(datapoints.json(), outfile)
 
 
+def save_live_logs(request, pods_containers_dict, patterns_to_log=None):
+    """
+    Save pods logs to files
+
+    """
+    tc_name = request.node.name
+
+    def finalizer():
+        """
+        Stop the thread that executed save_logs()
+
+        """
+        ocsci_config.RUN["save_logs"] = False
+        if thread:
+            thread.join()
+
+    request.addfinalizer(finalizer)
+
+    ocsci_config.RUN["save_logs"] = True
+
+    def save_logs():
+        while ocsci_config.RUN["save_logs"]:
```

Comment:
change it here to RUN.get("save_live_pod_logs")

Reply:
done